src/app/components/applications/SSV/MyAccount/components/Validator/FundingNewValidator/FundingNewValidator.tsx

```diff
@@ -30,14 +30,49 @@ const FundingNewValidator = () => {
   const process: SingleCluster = processStore.getProcess;
   const [checkedId, setCheckedId] = useState(0);
   const [depositSSV, setDepositSSV] = useState(0);
+  const [errorMessage, setErrorMessage] = useState({ text:'', link: { text:'', path:'' } });
+  const [showErrorMessage, setShowErrorMessage] = useState(false);
   const cluster = process.item;
   const newBurnRate = clusterStore.getClusterNewBurnRate(cluster, cluster.validator_count + 1);
-  const disableBtnCondition = depositSSV > ssvStore.walletSsvBalance || (depositSSV === 0 && checkedId === 2) || !checkedId;
+  const disableBtnCondition = (depositSSV === 0 && checkedId === 2) || !checkedId;
   const newRunWay = clusterStore.getClusterRunWay({
     ...cluster,
     burnRate: walletStore.toWei(parseFloat(newBurnRate.toString())),
     balance: walletStore.toWei(walletStore.fromWei(cluster.balance) + depositSSV),
   });
+  const calculateNewRunWayCondition = checkedId === 2 ? depositSSV > 0 : true;
+
+  useEffect(() => {
+    if (checkedId === 2 && depositSSV === 0) {
```

Comment:
Use constants
`OPTION_USE_CURRENT_BALANCE` := `1`
`OPTION_DEPOSIT_ADDITIONAL_FUNDS` := `2`

Use them in both logical and UI parts

Reply:
done